cmd/snap/cmd_changes.go

```diff
@@ -34,9 +34,12 @@ import (
 var shortChangesHelp = i18n.G("List system changes")
 var shortTasksHelp = i18n.G("List a change's tasks")
 var longChangesHelp = i18n.G(`
-The changes command displays a summary of the recent system changes performed.`)
+The changes command displays a summary of the recent system changes performed.
+`)
 var longTasksHelp = i18n.G(`
-The tasks command displays a summary of tasks associated to an individual change.`)
+The tasks command displays a summary of tasks associated to an
+individual change.
```

Comment:
Thanks for wrapping the help strings!

Reply:
Yeah... my thought was: `go-flags` wraps them at terminal end, but doesn't fold them so it's ugly. A lot of people that live in the terminal these days have 200+ wide terms which makes wrapping at terminal width really hard to read. Wrapping them at 78 makes it at least look ok unless you have a narrow terminal... so, there, done.

I'd be happiest if we wrapped at term width capped to ~100 (which seems to be the limit of legibility) plus indentation, but that's a pipedream for now.